src/main/main.cpp

```diff
@@ -67,7 +66,7 @@ int app_main(int argc, char* argv[])
     MemLeak memleak;
     base::SystemConsole systemConsole;
     app::AppOptions options(argc, const_cast<const char**>(argv));
-    she::ScopedHandle<she::System> system(she::create_system());
+    she::System* system = she::create_system();
```

Comment:
If this is no longer a ScopedHandle it won't be disposed in the end.
Use a unique_ptr/shared_ptr with a custom deleter instead?

Reply:
I guess we might want to keep ScopedHandle then.